src/Model/CustomerTokenService.php

```diff
@@ -67,6 +68,11 @@ class CustomerTokenService extends SourceCustomerTokenService
      */
     protected $requestThrottler;
 
+    /**
+     * @var UrlInterface
+     */
+    protected $urlBuilder;
```

Comment:
if for latest PWA, can have a strict class type 
`protected UrlInterface $urlBuilder;`